x/distribution/client/cli/query.go

```diff
@@ -10,82 +10,75 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	distr "github.com/cosmos/cosmos-sdk/x/distribution"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
 )
 
 // GetCmdQueryParams implements the query params command.
 func GetCmdQueryParams(queryRoute string, cdc *codec.Codec) *cobra.Command {
-	cmd := &cobra.Command{
+	return &cobra.Command{
 		Use:   "params",
-		Args:  cobra.ExactArgs(0),
+		Args:  cobra.NoArgs,
```

Comment:
Is there not a querier that returns _all_ params?

Reply:
I don't know. This is the code that @cwgoes wrote here.

Reply:
Going to dismiss this as not actionable.